### ISSUE TITLE
request.form returns list for values

### ISSUE THREAD

**Issue by sankaravadivel** [channelcat/sanic#288]
I am using voluptuous to validate form data which internally uses request.form['key'] notation to fetch values. When the form data is accessed that way we are getting a list containing the value instead of the value itself. It works as expected when we do request.form.get('key').

Thanks
Sankar

**Comment by seemethere** [comment]
That's because the implementation expects you to use `get`. It's a simple fix but idk how much it would effect.

**Comment by seemethere** [comment]
closed with #289

**Comment by LucasGrasso** [comment]
I had the exact opposite error. Was getting the first element of a array. Had to use 
```python
json.loads(str(req.form).replace("'", '"'))[
        "search_strings"
    ]
```

![image](https://user-images.githubusercontent.com/66641667/221483548-ba845cfe-a402-465d-8fc9-d5119b0c0491.png)
